### PR TITLE
chore(docs): document the `gas_price` RPC method

### PR DESCRIPTION
**Pull request by iTranscend**
Tracking issue: <https://github.com/near/near-jsonrpc-client-rs/issues/51>

Documented the `gas_price` RPC method, including easy-to-understand examples.

### CODE REVIEW

File: src/methods/gas_price.rs [hunk 1]
```diff
@@ -1,3 +1,57 @@
+//! Returns the gas price for a specific block_height or block_hash.
```
Comment (miraclx) [inline src/methods/gas_price.rs:1]:
```suggestion
//! Returns the gas price for a specific block height or block hash.
```

File: src/methods/gas_price.rs [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+//!
+//! ## Examples
+//!
+//! Returns the gas fees for this block:
+//! <https://explorer.near.org/blocks/6atGq4TUTZerVHU9qWoYfzXNBg3K4C4cca15TE6KfuBr>
```
Comment (miraclx) [inline src/methods/gas_price.rs:6]: this hash...

File: src/methods/gas_price.rs [hunk 3]
```diff
@@ -1,3 +1,57 @@
...
+//!     ```
+//!     # use near_jsonrpc_client::{methods, JsonRpcClient};
+//!     use near_primitives::types::BlockId;
+//!
+//!     # #[tokio::main]
+//!     # async fn main() -> Result<(), Box<dyn std::error::Error>> {
+//!     let client = JsonRpcClient::connect("https://archival-rpc.mainnet.near.org");
+//!
+//!     let request = methods::gas_price::RpcGasPriceRequest {
+//!         block_id: Some(BlockId::Height(48309771)),
```
Comment (miraclx) [inline src/methods/gas_price.rs:19]: ...isn't this height

**Review by miraclx — CHANGES_REQUESTED** [review]

File: src/methods/gas_price.rs (hunk 3 above)
Comment (iTranscend) [reply src/methods/gas_price.rs:19]: Yeah, this is height, the example for `hash` is below it

Comment (iTranscend) [reply src/methods/gas_price.rs:19]: Do you want the `hash` example to come first?

File: src/methods/gas_price.rs (hunk 2 above)
Comment (iTranscend) [reply src/methods/gas_price.rs:6]: Documented both `hash` and `height`, hence the URL.

I can change it to what's below if it'll make it look better

> Returns the gas fees for this [block](https://explorer.near.org/blocks/6atGq4TUTZerVHU9qWoYfzXNBg3K4C4cca15TE6KfuBr)

File: src/methods/gas_price.rs (hunk 3 above)
Comment (miraclx) [reply src/methods/gas_price.rs:19]: Both comments go together.

> This hash: `https://explorer.near.org/blocks/6atGq4TUTZerVHU9qWoYfzXNBg3K4C4cca15TE6KfuBr`
> Isn't this height: `48309771`

Comment (miraclx) [reply src/methods/gas_price.rs:19]: The height should be `61512623`.

Comment (iTranscend) [reply src/methods/gas_price.rs:19]: True, my bad

**Pull request merged**